apps/transport/lib/jobs/resource_unavailable_job.ex

```diff
@@ -41,24 +41,16 @@ end
 
 defmodule Transport.Jobs.ResourceUnavailableJob do
   @moduledoc """
-  Job checking if a resource is available over HTTP or not and
-  storing unavailabilities in that case.
-
-  It also updates the relevant resource and keeps up to the following fields:
-  - is_available (if the availability of the resource changes)
-  - url (if lastest_url points to a new URL)
+  This Job :
+  - Updates the url if needed by following the stable_url for files stored on data.gouv.fr
```

Comment:
```suggestion
  - Updates the URL if needed by following the latest_url for files stored on data.gouv.fr
```
`latest_url` est le nom de ce champ dans l'API de data.gouv.fr